### PR TITLE
[CI] remove unstable stateful tests

### PR DESCRIPTION
**Pull request by ZhiHanZ**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

will reintroduce it once runner run on public cloud

## Changelog

- Bug Fix

## Related Issues

Fixes #issue

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: scripts/ci/ci-run-stateful-tests-cluster.sh [hunk 1]
```diff
@@ -30,14 +30,14 @@ else
     exit 1
 fi
 
-./target/release/bendctl --databend_dir ./.databend --group local query ./tests/suites/0_stateful/ddl/create_table.sql
+./target/debug/bendctl --databend_dir ./.databend --group local query ./tests/suites/0_stateful/ddl/create_table.sql
```
Comment (BohuTANG) [inline scripts/ci/ci-run-stateful-tests-cluster.sh:33]: Why change to `debug`?

Comment (ZhiHanZ) [reply scripts/ci/ci-run-stateful-tests-cluster.sh:33]: to reduce ci compile time，similar to stateless tests

File: Makefile [hunk 2]
```diff
@@ -68,6 +68,8 @@ cross-compile-release:
 cli-build:
 	bash ./scripts/build/build-cli.sh build-cli
 
+cli-build-debug:
+	bash ./scripts/build/build-cli.sh build-cli-debug
```
Comment (BohuTANG) [inline Makefile:72]: It's better to make a break line

File: scripts/ci/ci-run-stateful-tests-cluster.sh (hunk 1 above)
Comment (PsiACE) [reply scripts/ci/ci-run-stateful-tests-cluster.sh:33]: How long does it take to run the stateful test in debug mode? Is the total time (compile + run) actually reduced?

Comment (BohuTANG) [reply scripts/ci/ci-run-stateful-tests-cluster.sh:33]: The debug mode in loading is slower than release, bendctl release compile time is more but the load data is less, I prefer to use release to do the stateful test.

Comment (ZhiHanZ) [reply scripts/ci/ci-run-stateful-tests-cluster.sh:33]: I shall use release mode once deployed runner into cloud service

**Comment by drmingdrmer** [comment]
What does the `unstable` mean?
Is there any problem with these two removed test?

**Comment by ZhiHanZ** [comment]
> What does the `unstable` mean? Is there any problem with these two removed test?

we are planning to use self-hosted runner to run stateful test, and deploy self hosted runner on EKS. self hosted runner is not ready, but we could run stateful test from local

**Pull request merged**